scripts/sort-imports.ts

```diff
@@ -341,20 +341,33 @@ async function processFile(filePath: string): Promise<void> {
       }
     }
     if (thisDeclaration) {
-      sortedLines[thisDeclaration.index] = DELETED_LINE;
-    }
-    if (thisDeclaration) {
-      let firstNonFileCommentLine = 0;
-      for (const line of sortedLines) {
-        if (line.startsWith("//!")) {
-          firstNonFileCommentLine++;
-        } else {
+      var onlyCommentsBeforeThis = true;
```

Comment:
`WaitGroup.zig` has a comment at the top of the file with attribution and license information. `sort-imports.ts` was incorrectly putting `const Self = @This()` before the comment because it wasn't a doc comment (`//!`), but attribution/license info shouldn't be a doc comment because it isn't relevant to the API.